plugins/FileHandlers/STLReader/STLReader.py

```diff
@@ -47,28 +31,11 @@ def __init__(self) -> None:
         )
         self._supported_extensions = [".stl"]
 
-    def load_file(self, file_name, mesh_builder, _use_numpystl = False):
-        file_read = False
-        if _use_numpystl:
-            Logger.log("i", "Using NumPy-STL to load STL data.")
-            try:
-                self._loadWithNumpySTL(file_name, mesh_builder)
-                file_read = True
-            except:
-                Logger.logException("e", "Reading file failed with Numpy-STL!")
-
-        if not file_read:
-            Logger.log("i", "Using legacy code to load STL data.")
-            f = open(file_name, "rb")
-            if not self._loadBinary(mesh_builder, f):
```

Comment:
The `_loadBinary` and `_loadAscii` functions are no longer used now, and should be removed.